src/org/thoughtcrime/securesms/InstantOnboardingActivity.java

```diff
@@ -467,12 +468,16 @@ private void startQrAccountCreation(String qrCode)
 
     DcHelper.getEventCenter(this).captureNextError();
 
-    if (!dcContext.setConfigFromQr(qrCode)) {
-      progressError(dcContext.getLastError());
-      return;
-    }
-    DcHelper.getAccounts(this).stopIo();
-    dcContext.configure();
+    new Thread(() -> {
```

Comment:
I think it is better to use `Util.runOnBackground()` or  `Util.runOnAnyBackgroundThread()`

Reply:
i am always unsure about these options. we're using the java-nly approach also ar other places.

which one is better? and why?

Reply:
I am not sure, but since this was created in utils and used in other places in a similar way (runOnBackground + inside runOnMainThread() at some point) I guess is better to use the same approach everywhere, in any case what runOnBackground does can then be changed according to what is "better" unlike manually creating the thread and calling start()

Reply:
ftr, `runOnBackground()` uses `AsyncTask.THREAD_POOL_EXECUTOR` which is marked as deprecated ("Using a single thread pool for a general purpose results in suboptimal behavior for different tasks [...] Use or create an Executor configured for your use case" - i think, real differences do only pop up when really using threads massively, in our case it probably just does not matter.